src/partition/InputLines.cc

```diff
@@ -210,7 +243,8 @@ class InputLines::Impl {
 
     size_t const _blockSize;
     bool const _skipFirstLine;
-
+    const std::vector<std::string> _paramNames;
```

Comment:
Incorrect coding style. `const` should always follow the base type. The correct way of declaring the member is:
```
std::vector<std::string> const _paramNames;
```
Please, see the relevant section of the LSST C++ Style Guide: https://developer.lsst.io/cpp/style.html#the-const-keyword-should-be-listed-after-the-type-name

Reply:
paramNames has been replaced by a struct object that contains more csv formating parameters